style/authentication/README.md

```diff
@@ -0,0 +1,27 @@
+# Authentication
+
+When building applications that use authentication, security is crucial. To help ensure that we are building secure applications, we follow the [OWASP Authentication General Guidelines](https://www.owasp.org/index.php/Authentication_Cheat_Sheet#Authentication_General_Guidelines).
+
+Because security is a constantly moving target, we recommend that you [don't roll your own authentication](https://blog.codeship.com/why-you-shouldnt-roll-your-own-authentication/), and instead rely on a battle-tested authentication mechanism. As an example, for Rails apps we use [Devise](https://github.com/plataformatec/devise).
+
```

Comment:
for Javascript/React things, we've also used [Passport JS](http://passportjs.org/) if you want to add that in as a resource.